SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestRepNotify/SpatialTestRepNotify.cpp

```diff
@@ -162,66 +210,51 @@ void ASpatialTestRepNotify::PrepareTest()
 			bOldArrayShouldHaveTwoElements = bOldArrayShouldHaveTwoElements || GetNetDriver()->IsA(USpatialNetDriver::StaticClass());
 			if (bOldArrayShouldHaveTwoElements)
 			{
-				if (OldTestArray.Num() != 2)
-				{
-					FinishTest(EFunctionalTestResult::Failed, TEXT("OnRepTestArray should have been called with 2 entries in the old Array "
-																   "on Spatial or in Native on 4.26 and above"));
-					return;
-				}
+				RequireEqual_Int(TestActor->OldTestArray.Num(), 2,
+								 TEXT("OnRepTestArray should been called with 2 entries in the old Array "
+									  "on Spatial or in Native on 4.26 and above"));
 			}
 			else
```

Comment:
I feel like using an else then a subsequent if here is clearer as to intention of what we are checking, but could just be an `else if` - thoughts?

Reply:
I like it as is, but either would be acceptable imho.